package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "next-contentful",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
```

```suggestion
    "start": "npx serve@latest ./out",
```

静的エクスポートで `next start` は使えないようなので、上のようなコマンドが推奨されてました。